templates/base.html

```diff
@@ -49,6 +49,10 @@
         {% include '_includes/feeds.html' %}
         {% endblock feed_links %}
         {% include '_includes/analytics.html' %}
+        {% if (article and article.photo_gallery) %}
```

Comment:
Why are we adding this in base.html? this plugin depends on `article` variable, right?

Then all the scripts and CSS related to `article` should go into `article.html` template.

Reply:
According  to https://github.com/getpelican/pelican-plugins/tree/master/photos it instructs to place it in 'base'

Initially the loop is for this:

{% if (article and article.photo_gallery) or (articles_page and articles_page.object_list[0].photo_gallery) %}

But I was getting errors on the articles_page.object_list so initially disabled it any idea here?

Reply:
It should work even if you put it in the head of article template. The only requirement is to put it in the head. We already do it for title and meta tags. 

`articles_page` this is for pagination. We don't need it.

Please move into article.html.

Reply:
Seems that the problem is that the areas available in article.html don't render 'after' other libraries are loaded, so it loads first this one without jquery so then, it fails.

How can I tell this one to appear just before </head> ?

Reply:
Probably with this we can move it to article.html and get rid of code in base.html so any advice is appreciated here, thanks